packages/sdk/contractkit/src/wrappers/Election.ts

```diff
@@ -355,14 +355,29 @@ export class ElectionWrapper extends BaseWrapperForGoverning<Election> {
     )
   }
 
+  /**
+   * Creates a transaction object for revoking active votes.
+   * @dev In cases where `revokeActive` transaction object will be executed right after
+   * a `revokePending` transaction object, one must use the total revoke amount to account
+   * for any removed pending votes when obtaining the `findLesserAndGreaterAfterVote`.
+   * @param value Amount to be removed from active votes.
+   * @param totalValue Amount used to obtain the lesser and greater addresses.
```

Comment:
Agreed that `totalValue` is a somewhat confusing parameter. Potential better options:
* `pendingRevokedValue` (so pass in the amount you know you will revoke but haven't submitted to the blockchain yet, rather than the total of the current revoke + pending revoke)
* Pass in `lesser` and `greater` as optional parameters. If passed in, don't calculate here, otherwise compute based on `value`. Then in `revoke` we'd just compute them manually based on known total.